src/BuildValues.props

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Thanks Jason - this file is automatically generated and I reverted my changes in the most recent commit.  @dotnet-bot will update it as part of our official build.